tests/cli-misc.rs

```diff
@@ -568,6 +568,40 @@ fn rename_rls_remove() {
     });
 }
 
+#[test]
+fn install_with_profile() {
+    clitools::setup(Scenario::Full, &|config| {
```

Comment:
If you'd put this in `cli-v2.rs` and copied, say, `test_complete_profile_skips_missing_when_forced()` then you'd have seen `setup_complex()` which has what you wanted here, without using a different pattern.

Even if we keep this test body, I'd prefer the test were in `cli-v2.rs`